examples/terraform-submodules/gke/module.tf

```diff
@@ -53,9 +53,14 @@ variable "enable_image_streaming" {
   default = "true"
 }
 
-variable "zone" {
```

Comment:
We also need to update the [website page](https://github.com/googleforgames/agones/blob/main/site/content/en/docs/Installation/Terraform/gke.md) to indicate the change. Noted for changes to the website, we generally gate them so that they do not appear until the next release. Even though this doesn't require new features in Agones, it still makes sense to wait to publish this guidance until our next release.
https://agones.dev/site/docs/contribute/#within-a-page describes how to add the feature gate tags to the website pages.

Reply:
Done!